llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -631,6 +631,48 @@ define void @foo(i1 %c0, i8 %v0, i8 %v1, i1 %c1) {
   }
 }
 
+TEST_F(SandboxIRTest, ExtractElementInst) {
+  parseIR(C, R"IR(
+define void @foo(<2 x i8> %vec, i32 %idx) {
+  %ins0 = extractelement <2 x i8> %vec, i32 %idx
+  ret void
+}
+)IR");
+  Function &LLVMF = *M->getFunction("foo");
+  sandboxir::Context Ctx(C);
+  auto &F = *Ctx.createFunction(&LLVMF);
+  auto *ArgVec = F.getArg(0);
+  auto *ArgIdx = F.getArg(1);
+  auto *BB = &*F.begin();
+  auto It = BB->begin();
+  auto *Ins0 = cast<sandboxir::ExtractElementInst>(&*It++);
+  auto *Ret = &*It++;
+
+  EXPECT_EQ(Ins0->getOpcode(), sandboxir::Instruction::Opcode::ExtractElement);
+  EXPECT_EQ(Ins0->getOperand(0), ArgVec);
+  EXPECT_EQ(Ins0->getOperand(1), ArgIdx);
+  EXPECT_EQ(Ins0->getVectorOperand(), ArgVec);
+  EXPECT_EQ(Ins0->getIndexOperand(), ArgIdx);
+  auto *NewI1 =
```

Comment:
I think we need a test for `getVectorOperandType()`. Something like:
```
EXPECT_EQ(EI->getVectorOperandType(), ArgVec->getType());
```

Reply:
Done!